test/e2e/arktos/multi_tenancy/test_suites/multi_tenancy_controller/test_deployment_replicaset_controller.yaml

```diff
@@ -0,0 +1,141 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Deployment & ReplicaSet Controller Tests ~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
like this BDD style: easy to understand; end user should be able to add test cases without problem.

Reply:
:-)